adafruit_portalbase/__init__.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         json_path=None,
         regexp_path=None,
         json_transform=None,
+        success_callback=None,
```

Comment:
I like this new `success_callback` parameter, this can make the helper class more versatile because user code can customize what happens after fetch more.

We should add a docstring entry for it in the list above I think.

Reply:
Sounds good.